TileStache/Goodies/VecTiles/transform.py

```diff
@@ -943,6 +945,10 @@ def exterior_boundaries(feature_layers, base_layer,
     """
     layer = None
 
+    # don't start processing until the start zoom
+    if zoom < start_zoom:
+        return layer
+
```

Comment:
Ah, I didn't realize that! I like the start_zoom being 0 explicitly though if unspecified, so thanks for updating that.